From 63eefdd50f1cf53c6dc46c83dce1441df58924fd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Sun, 25 Apr 2021 03:17:39 +0200
Subject: arm64: dts: sun50i-a64-pinephone: Power off the touch controller in
 sleep

Otherwise it stays on and consumes quite a bit of power.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index b44f1ce14e06..e39b070deec9 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -484,6 +484,7 @@ touchscreen@5d {
 		VDDIO-supply = <&reg_ldo_io0>;
 		touchscreen-size-x = <720>;
 		touchscreen-size-y = <1440>;
+		poweroff-in-suspend;
 	};
 };
 
-- 
2.35.3

